lib/inflections/sv.rb

```diff
@@ -0,0 +1,93 @@
+# encoding: utf-8
+#
+# Swedish inflections
+#
+# Original rules by olabini - https://olabini.com/blog/2006/08/announcing-swedish-rails
+# Adapted by richardjohansson - for https://github.com/mynewsdesk/swedish_pluralize
+#
+module Inflections
+  ActiveSupport::Inflector.inflections(:sv) do |inflect|
+    inflect.clear
+
+    # --- Plural ---
+
+    # -er
+    inflect.plural(/([tmnslpr])$/i, '\1er') # produkt, produkter
+
+    # -or
+    inflect.plural(/a$/i, 'or') # flicka, flickor
+
+    # -ar
+    inflect.plural(/e$/i, 'ar') # pojke, pojkar
+    inflect.plural(/g$/i, 'gar')
+    inflect.plural(/l$/i, 'lar') # bil, bilar
+    inflect.plural(/el$/i, 'lar') # fågel, fåglar
+
+    # -er
+    inflect.plural(/kel$/i, 'kler') # muskel, muskler
+    inflect.plural(/tel$/i, 'tlar') # titel, titlar
+
+    # -r
+    inflect.plural(/de$/i, 'der') # fiende, fiender
+
+    # -n
+    inflect.plural(/le$/i, 'lem') # muskel, muskler
```

Comment:
Yep, poor example. # muskeln, musklerna would be
`inflect.plural(/n$/i, 'lerna') # muskeln, musklerna`